jiant/tasks/tasks.py

```diff
@@ -2700,6 +2700,106 @@ class MultipleChoiceTask(Task):
     pass
 
 
+@register_task("SociallQA", rel_path="SociallQA/")
```

Comment:
maybe I'm crazy, but are these currently spelled as "Social*l*QA" and not "Social*I*QA" (capital I vs lowercase L)